### PR TITLE
Added login functionality

### PR DESCRIPTION
**Pull request by kratikasukhwal99**
Added login funationality and fixed [bug id]

### CODE REVIEW

File: FirebaseConfig/stagedev/GoogleService-Info.plist [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>368901996753-ltbuu6phjaa8uv2h7gumssh1mkpl3f7s.apps.googleusercontent.com</string>
```
Comment (manishpathak3173) [inline FirebaseConfig/stagedev/GoogleService-Info.plist]: Please remove this tag

**Review by manishpathak3173 — CHANGES_REQUESTED** [review]
@kratika-17 Please remove the tag.

File: FirebaseConfig/stagedev/GoogleService-Info.plist (hunk 1 above)
Comment (kratikasukhwal99) [reply FirebaseConfig/stagedev/GoogleService-Info.plist]: @manishpathak3173 I've made the changes , Please review again.

**Review by manishpathak3173 — APPROVED** [review]
Approved.

**Pull request merged**